x/wasm/lbm/types/tx_test.go

```diff
@@ -0,0 +1,122 @@
+package types
+
+import (
+	sdk "github.com/line/lbm-sdk/types"
+	wasmTypes "github.com/line/lbm-sdk/x/wasm/types"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"testing"
```

Comment:
```suggestion
	"testing"

	"github.com/stretchr/testify/assert"
	"github.com/stretchr/testify/require"

	sdk "github.com/line/lbm-sdk/types"
	wasmTypes "github.com/line/lbm-sdk/x/wasm/types"
```

In most cases, imports follow the order of golang native modules, then other repo, then lbm-sdk.